Task-7/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_7
+{
+    class Program
+    {
+        public static float f(float x)
```

Comment:
Методы в C# принято называть с большой буквы. Для работы с числами с плавающей тосчкой мы договорились, если никаких особых причин нет, использовать формат `double`
```suggestion
        public static double F(double x)
```